java/src/jmri/jmrix/loconet/LnTurnoutManager.java

```diff
@@ -205,7 +206,7 @@ public int getBitFromSystemName(String systemName) {
             log.error("invalid character in header field of loconet turnout system name: {}", systemName);
             return (0);
         }
-        // name must be in the LTnnnnn format (L is user configurable)
+        // name must be in the LiTnnnnn format (Li is user configurable)
```

Comment:
Perhaps this would better say "Name must start with the current system prefix followed by a number"

Reply:
fixed, like NCE